backend/src/src-vendors/src/main/java/org/eclipse/sw360/vendors/VendorHandler.java

```diff
@@ -34,28 +32,28 @@
  */
 public class VendorHandler implements VendorService.Iface {
 
-    private final VendorRepository vendorRepository;
+    private final VendorDatabaseHandler vendorDatabaseHandler;
```

Comment:
I think `handler` would be a good name for this field. Other `Handler`s call their `DatabaseHandler`s like that.